src/modules/commentform.js

```diff
@@ -0,0 +1,6 @@
+const commentform = () => (`<h4>Add a comment</h4>
+    <input type="text" placeholder="Your name" required>
+    <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Your Insights" required></textarea>
+    <button class="popup-btn">Comment</button>`);
+
```

Comment:
- Kindly the comments are working as expected however it redirects to the home page instead of adding the comment to the comment list directly, please fix it to fulfill the requirement and have a better user experience :+1: 
![Screenshot from 2023-04-21 19-43-08](https://user-images.githubusercontent.com/52010789/233690378-94c7639e-9876-40d2-8bd7-4e0b1c51e0b8.png)

Reply:
Fixed